lib/BaseProjectLibrary.ts

```diff
@@ -62,6 +62,10 @@ export class BaseProjectLibrary implements ProjectLibrary {
 				.filter(x => x !== this._projectsPath && x !== this._customTemplatesPath);
 
 			for (const componentFolder of componentFolders) {
+				if (componentFolder === "generate") {
```

Comment:
There's a filter right above that does the same thing, you can add that exclusion there.